src/autofix/fixer.rs

```diff
@@ -24,17 +23,19 @@ impl From<bool> for Mode {
 }
 
 /// Auto-fix errors in a file, and write the fixed source code to disk.
-pub fn fix_file(checks: &mut [Check], contents: &str, path: &Path) -> Result<()> {
-    if checks.iter().all(|check| check.fix.is_none()) {
-        return Ok(());
-    }
```

Comment:
this is probably a bad idea 🤔 Like maybe we want to special case stdout here and only print if it's stdout, otherwise `--fix` will always rewrite all files. I'll revisit this tomorrow.

Reply:
Ended-up returning just `Option<string>` now that we're not writing the file, there is no error possible :D